packages/custom-config-poc-app/amplify/backend.ts

```diff
@@ -0,0 +1,254 @@
+import { defineBackend } from '@aws-amplify/backend';
+import { myFunc } from './function';
+import * as apigateway from 'aws-cdk-lib/aws-apigateway';
+import * as process from 'process';
+import { auth } from './auth/resource';
+import * as location from 'aws-cdk-lib/aws-location';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import { PlaceIndex } from '@aws-cdk/aws-location-alpha';
+import { CfnOutput } from 'aws-cdk-lib';
+import { ClientConfigFormat } from '@aws-amplify/client-config';
+
+const backend = defineBackend({
+  auth,
+  myFunc,
+});
+
+// ####### API Gateway + Lambda Start #######
+
+const apiGatewayStack = backend.createStack('APIGateway');
```

Comment:
Why create a new stack? Why not reuse the stack with something like `backend.myFunc.resources.lambda.stack`?

I worry that using a new named stack might conflict between sandboxes / amplify-deployed cdk?

Reply:
This code isn't going to be part of solution.
It's just piece of POC Gen2 app that I used for testing.
Both approaches are possible.